### PR TITLE
Announcement of Ruby 2.1.10 release.

### PR DESCRIPTION
**Pull request by unak**
Please review my poor English.

Note that this PR does not include `_config.yml`.
It's intentional because 2.1.10 is for testing, not for production.

### CODE REVIEW

File: en/news/_posts/2016-04-01-ruby-2-1-10-released.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+layout: news_post
+title: "Ruby 2.1.10 Released"
+author: "usa"
+translator:
+date: 2016-04-01 02:00:00 +0000
+lang: en
+---
+
+Ruby 2.1.10 has been released.
+This release is not for production environment, but for compatibility test with two-digit version number.
```
Comment (stomar) [inline en/news/_posts/2016-04-01-ruby-2-1-10-released.md]: This release is not intended for production use, but for compatibility tests with two-digit version numbers.

File: en/news/_posts/2016-04-01-ruby-2-1-10-released.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+You don't have to replace Ruby 2.1.9 by 2.1.10 in normal use.
+
+[As announced in 2.1.9 release](https://www.ruby-lang.org/en/news/2016/03/30/ruby-2-1-9-released/), Ruby 2.1.10 does not include any changes from 2.1.9, except for its version number. (And, this includes only one small change in its test suite.)
```
Comment (stomar) [inline en/news/_posts/2016-04-01-ruby-2-1-10-released.md]: As announced in the 2.1.9 release post

File: en/news/_posts/2016-04-01-ruby-2-1-10-released.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Please test your applications and/or libraries for testing compatibility with two-digit version number.
```
Comment (stomar) [inline en/news/_posts/2016-04-01-ruby-2-1-10-released.md]: for <del>testing</del> compatibility with two-digit version number**s**.

**Comment by unak** [comment]
@stomar Thank you! applied.

**Comment by hsbt** [comment]
@unak Do you ready to publish this announcement?

**Comment by unak** [comment]
@hsbt I'm planning to publish at 11:00 am JST. (see date field)

**Comment by hsbt** [comment]
@unak I see. Thanks!

**Pull request merged**